Blog Post: World Ocean Day

steffilazerte opened the pull request:
Draft
* [X] I have read the Content Guidelines.
* [X] I have read the Technical Guidelines.
* [X] I used or followed the R Markdown or Markdown template.
* [X] I have followed the Style Guide.
* [X] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [X] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.

steffilazerte commented:
Okay @stefaniebutland, I'm ready for a review! I had to switch out robis in the end, for rinat because robis isn't actually rOpenSci 🤦🏼 

@sckott could you do a quick review too? For the use of your rnoaa?

The second section might be a bit wishy washy as I changed things over last minute (I need some space from this for a while!)

Thanks!

sckott commented:
Looks good @steffilazerte 

- Just saw this the other day from mike sumner https://github.com/AustralianAntarcticDivision/seaice maybe worth mentioning in notes, up to you
- mike references in the above repo this ropensci blog post https://ropensci.org/blog/2018/11/13/antarctic/ that talks about sea ice data

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+## Sea ice
+
+One of the biggest challenges for protection our ocean is climate change[^2].
```
```suggestion
One of the biggest challenges for protecting our ocean is climate change[^2].
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+Last September the Arctic sea ice had the 2nd-lowest extent in a 40-year record[^3]. 
+This isn't good for the Arctic ecosystem and has impacts not only on wildlife but on humans as well [^4]. `r emo::ji("sad")`
+
+Let's see if we can't explore ice extent ourselves using open data and open source packages part of the [rOpenSci package collections](/packages).
```
```suggestion
Let's see if we can't explore ice extent ourselves using open data and open source packages that are part of the [rOpenSci package collections](/packages).
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+Let's see if we can't explore ice extent ourselves using open data and open source packages part of the [rOpenSci package collections](/packages).
+
+Here, we'll use two rOpenSci packages
+- [rnoaa](https://docs.ropensci.org/rnoaa) to access NOAA data on sea ice coverage
```
can you add somewhere, something like
rnoaa by Scott Chamberlain for rOpenSci and rnaturalearth by Andy South, peer reviewed by rOpenSci

steffilazerte replied:
Done

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+(Both these packages can do so much more, so check out the docs if you're curious)
+
+We'll also use sf for working with spatial data, dplyr for data manipulation, and ggplot2 for plotting. 
```
pondering whether it's good to link to these pkgs

steffilazerte replied:
Done

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
````diff
@@ -0,0 +1,333 @@
+ice <- sea_ice(year = seq(1980, 2020, 5), month = "Sep", pole = "N")
+```
+
+`ice` is a list, which each list item corresponding to a year.
````
```suggestion
`ice` is a list, with each list item corresponding to a year.
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+Each year is a 'fortified' data frame which you could plot using `geom_polygon()`. 
+But I want to get the projection right, so let's bind the rows together (adding year as a parameter) and convert to an sf object. 
+We'll transform to EPSG 3411 which refers to the NSIDC Sea Ice Polar Stereographic North projection commonly used to map the Arctic.
```
can you link to this for folks unfamiliar?

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
````diff
@@ -0,0 +1,333 @@
+  geom_sf(data = ice_sf, aes(fill = year), colour = NA)
+```
+
+Oooo very nice! But a couple of points could be improved
````
I'd appreciate ~ 1 sentence to help reader (me) quickly interpret what the map/plot is telling me

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+*[Andrew Shiva / Wikipedia / CC BY-SA 4.0](https://commons.wikimedia.org/wiki/File:Antarctic_Sound-2016-Brown_Bluff%E2%80%93Leopard_seal_(Hydrurga_leptonyx)_04.jpg)*
+
+
+In addition to rnaturalearth, sf, dplyr, and ggplot2 which we loaded above, we'll use lubridate to handle date/times, and the rOpenSci package [rinat](https://docs.ropensci.org/rinat), for accessing species observations from the citizen science project [iNaturalist](https://www.inaturalist.org/).
```
again pls mention pkg author/maintainer 
and tag folks when you tweet

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
````diff
@@ -0,0 +1,333 @@
+glimpse(seals)
+```
+
+Alright, we have 59 observations around Antarctica, let's see where they're from.
````
```suggestion
Alright, we have 59 observations of leopard seals around Antarctica, let's see where they're from.
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+**[rinat](https://docs.ropensci.org/rinat)**
+
+- [2014-03-26 rOpenSci blog "Accessing iNaturalist data](/blog/2014/03/26/rinat/)
```
```suggestion
- [2014-03-26 rOpenSci blog "Accessing iNaturalist data"](/blog/2014/03/26/rinat/)
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+**[rnoaa](https://docs.ropensci.org/rnoaa)**
+
+- [2014-03-13 rOpenSci blog "rnoaa - Access to NOAA National Climatic Data Center data"](/blog/2014/03/13/rnoaa/)
```
list newer post first?

stefaniebutland replied:
If you see any challenges with linking to such old posts, maybe note that they're old?

steffilazerte replied:
I changed the order, and didn't mention problems. That's why the dates are there, is should (hopefully) be apparent

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+
+- [2014-03-26 rOpenSci blog "Accessing iNaturalist data](/blog/2014/03/26/rinat/)
+
+## Have your own use cases?
```
```suggestion
## Have you used rOpenSci packages to study our ocean?
```

stefaniebutland commented on content/blog/2021-06-08-world-ocean-day/index.Rmd:
```diff
@@ -0,0 +1,333 @@
+- [2014-03-26 rOpenSci blog "Accessing iNaturalist data](/blog/2014/03/26/rinat/)
+
+## Have your own use cases?
+We'd love to share them!
```
```suggestion
We'd love to share your examples!
```

stefaniebutland reviewed (commented):
I LOVE the post. tone, flow, technical but inviting to anyone.

this is brilliant esp with the image after
![Screen Shot 2021-06-07 at 9 25 43 AM](https://user-images.githubusercontent.com/11927811/121057286-f7f30f00-c773-11eb-8678-b4a9a89f67aa.png)


Before the call to action for use cases could you include a list of other rOpenSci pkgs that can/have been used to study ocean stuff?
worrms, mregions, https://docs.ropensci.org/antanym/

And and add a sentence (similar spot near end) that this post by Ben Raymond and Michael Sumner on behalf of the Antarctic rOpenSci community shares a use case ....blah blah sea ice and ocean depth ... shows the path of an elephant seal ....https://ropensci.org/blog/2018/11/13/antarctic/

steffilazerte commented:
Thanks @stefaniebutland and @sckott 

In addition the the specific comments above:
- I added references to the seaice package
- I added a new section for other rOpenSci resources including the blog post and other packages

steffilazerte commented:
Draft Tweet

---

[blog] 🎉 Celebrating World Ocean Day rOpenSci Style 🎉 by @steffilazerte

🐠🌊🐬🦭🐙🐟 Today is World Ocean Day! 🐟🐙🦭🐬🌊🐠

Let's celebrate and honour the Ocean which connects us all by exploring some themes with rOpenSci packages

- 🧊 Arctic sea ice changes over the years (#rnaturalearth and #rnoaa)
- 🦭Leopard seal observations in the Antarctic from iNaturalist (#rnaturalearth and #rinat)

https://ropensci.org/blog/2021/06/08/world-ocean-day/

#rstats #worldoceanday

(1/2)

---

🙏 Thanks to the awesome package authors and maintainers:
@southmapr (rnaturalearth), @sckottie (rnoaa for rOpenSci), Edmund Hart and Stéphane Guillou (rinat)

(2/2)
